src/Api/CollectorsTrait.php

```diff
@@ -238,7 +238,7 @@ public function sendCollectorMessage($collectorId, $messageId, \DateTime $schedu
 	{
 		return $this->sendRequest(
 			$this->createRequest('POST', sprintf('collectors/%s/messages/%s/send', $collectorId, $messageId), [
-				'query' => [ 'scheduled_date' => $scheduledDate ? $scheduledDate->format() : null ]
+				'query' => [ 'scheduled_date' => $scheduledDate ? $scheduledDate->format(DATE_ATOM) : date(DATE_ATOM) ]
```

Comment:
For some reason if you send null to surveymonkey API, they do not like it. But then again, they also say that it is not required in their documentation and not tell us what it defaults to if we don't provide one. I am really confused. I just made it so it defaults not whatever the time is a the time of execution.